docs/user-guide/scheduled-jobs.md

```diff
@@ -0,0 +1,155 @@
+---
+assignees:
+- erictune
+- soltysh
+- janetkuo
+
+---
+
+* TOC
+{:toc}
+
+## What is a _Scheduled Job_?
+
+A _Scheduled Job_ manages time based [Jobs](/docs/user-guide/jobs/), namely:
+
+* Once at a specified point in time
+* Repeatedly at a specified point in time
+
+One ScheduledJob object is like one line of a _crontab_ (cron table) file. It runs a job periodically
+on a given schedule, written in [Cron](https://en.wikipedia.org/wiki/Cron) format.
+
+A typical use case is:
+
+* Schedule a job execution at a given point in time.
+* Create a periodic job, e.g. database backup, sending emails.
+
+### Prerequisites
+
+You need a working Kubernetes cluster at version >= 1.4, with batch/v2alpha1 API turned on by passing
+`--runtime-config=batch/v2alpha1` while brining up the API server (see [Turn on or off an API version
+for your cluster](/docs/admin/cluster-management/#turn-on-or-off-an-api-version-for-your-cluster) for
+more). You cannot use Scheduled Jobs on a hosted Kubernetes provider that has disabled alpha resources.
+
+## Creating a Scheduled Job
+
+Here is an example Scheduled Job. Every minute, it runs a simple job to compute π to 2000 places and prints it out.
+
+{% include code.html language="yaml" file="sj.yaml" ghlink="/docs/user-guide/sj.yaml" %}
+
+Run the example scheduled job by downloading the example file and then running this command:
+
+```shell
+$ kubectl create -f ./sj.yaml
+scheduledjob "pi" created
+```
+
+Alternatively, use `kubectl run` to create a scheduled job without writing full config:
+
+```shell
+$ kubectl run pi --schedule="0/1 * * * ?" --image=perl --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'
+scheduledjob "pi" created
+```
+
+After creating the scheduled job, get its status using this command:
+
+```shell
+$ kubectl get scheduledjob pi
+NAME      SCHEDULE      SUSPEND   ACTIVE    LAST-SCHEDULE
```

Comment:
`LAST-SCHEDULE` sounds like a previous value of `SCHEDULE` but it isn't.
Is there another name that is short and conveys the meaning?